rotkehlchen/logging.py

```diff
@@ -206,3 +206,18 @@ def configure_logging(args: argparse.Namespace) -> None:
         logging.getLogger('substrateinterface.base').setLevel(logging.CRITICAL)
         logging.getLogger('eth_hash').setLevel(logging.CRITICAL)
         logging.getLogger('vcr').setLevel(logging.CRITICAL)
+
+
+log = RotkehlchenLogsAdapter(logging.getLogger(__name__))
+
+
+def enter_exit_debug_log(name: str | None = None) -> 'Callable':
```

Comment:
We can use this in many other places no?

Reply:
yeah, I'll add this in all those places after this PR

Reply:
https://github.com/rotki/rotki/pull/7635